libp2p/peer/peermetadata_interface.py

```diff
@@ -16,7 +23,7 @@ def get(self, peer_id, key):
         """
 
     @abstractmethod
-    def put(self, peer_id, key, val):
+    def put(self, peer_id: ID, key: Any, val: Any) -> None:
```

Comment:
https://github.com/libp2p/go-libp2p-core/blob/master/peerstore/peerstore.go#L80-L86
```suggestion
    def put(self, peer_id: ID, key: str, val: Any) -> None:
```